scripts/sdk-coin-generator/index.js

```diff
@@ -174,11 +175,30 @@ module.exports = class extends Generator {
     addNewCoinToBitgo(this.contextRoot, this.answers);
     addNewCoinToBitgoTsConfig(this.contextRoot, this.answers);
 
-    await this.addDependencies({
-      '@bitgo/sdk-core': '^1.0.1',
-    });
+    switch (this.answers.boilerplate) {
+      case 'utxo':
+        await this.addDependencies({
+          '@bitgo/abstract-utxo': '^1.0.0',
```

Comment:
can these versions be resolved from the current package.json files instead of hard coding them?

Reply:
Yeah, I can go fetch those from the files.